backend/mora/service/address.py

```diff
@@ -312,14 +314,44 @@ def get_one_address(c, addrrel, class_cache=None):
     if scope == 'DAR':
         # unfortunately, we cannot live with struktur=mini, as it omits
         # the formatted address :(
-        r = session.get(
-            'http://dawa.aws.dk/adresser/' + addrrel['uuid'],
-            params={
-                'noformat': '1',
-            },
-        )
+        try:
+            r = session.get(
+                'https://dawa.aws.dk/adresser/' + addrrel['uuid'],
+                params={
+                    'noformat': '1',
+                },
+            )
+        except Exception as exc:
+            # the exception above is overly broad for a) safety and b)
+            # testing -- specifically, the exception raised by
+            # requests_mock does not descend from RequestException :(
+            util.log_exception('failed to get address ' + addrrel['uuid'])
+
+            return {
```

Comment:
We build this object four different places now. Maybe it should be extracted to a function, to ensure consistency of the objects being created?